tests/rptest/services/redpanda.py

```diff
@@ -1428,6 +1431,22 @@ def stop(self, **kwargs):
         Override default stop() to execude stop_node in parallel
         """
         self._stop_time = time.time()  # The last time stop is invoked
+        self.logger.info("%s: exporting cluster config" % self.who_am_i())
+
+        service_dir = os.path.join(
+            TestContext.results_dir(self._context, self._context.test_index),
+            self.service_id)
+        cluster_config_filename = os.path.join(service_dir,
+                                               "cluster_config.yaml")
+        self.logger.debug("%s: cluster_config_filename %s" %
+                          (self.who_am_i(), cluster_config_filename))
+
+        if not os.path.isdir(service_dir):
+            mkdir_p(service_dir)
```

Comment:
@andrewhsu any reason you didn't use the `RedpandaService.logs` feature that can be used to collect data (e.g. we use that to collect logs, backtraces, coverage, executables, etc...) ?

Reply:
@dotnwat it seemed like the `RedpandaService.logs` would collect data per node. Since the cluster properties should be the same for each node in a cluster, I assumed we only needed to get the cluster config once and therefore I calculated the `service_dir` path (the parent dir of each node dir) to place the `cluster_config.yaml` within.

let me know if there is something i'm missing and i can make adjustments in a follow-up PR

Reply:
that sounds logical to me. thanks for the explanation!